ZorroOnt/README.md

```diff
@@ -0,0 +1,10 @@
+Persistent identifiers for the ZorroOntology.
+I have created an ontology specification draft page for my ontology, with this URI 
+"https://kai-vu.github.io/zorro/ZorroOntologySpec/doc/index-en.html"
+I would like to redirect it to more user friendly URI by removing the "index-en.html"
```

Comment:
- The text here doesn't need to explain what is happening.  Could remove this.
- To be clear, the `.htaccess` (as of this comment) is going to redirect `https://w3id.org/ZorroOnt/[...]` to `https://kai-vu.github.io/zorro/[...]`.  Did you want to add that `ZorroOntologySpec/doc/index-en.html` part to the target?  There are various ways to do this depending on what you are trying to do.  Ideally you would test the rules here do what you intend.
- That target does doc was generated with an invalid URL `http://www.ZorroUpperLevelOntology/`.  That should be fixed at some point.